File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("Height of the pyramid?: ");
    }
    while (n < 1 || n > 8);

    for (int i = 1; i <= n; i++)
    {
        for (int j = 0; j < n - i; j++)
        {
            printf(" ");
        }

        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50

while True:
    try:
        height = cs50.get_int("Height of the pyramid? ")
        if 1 <= height <= 8:
            break
    except ValueError:
        pass

for i in range(1, height + 1):
    spaces = " " * (height - i)
    hashes = "#" * i
    print(spaces + hashes + " " + hashes)

for i in range (height - 1, 0, -1):
    spaces = " " * (height - i)
    hashes = "#" * i
    print(spaces + hashes + " " + hashes)
